threadbare/operations.py

```diff
@@ -383,6 +384,21 @@ def single_command(cmd_list):
     return " && ".join(map(str, cmd_list))
 
 
+def prompt(msg):
+    """issues a prompt for input. 
+    raises a PromptedException if `abort_on_prompts` in `state.ENV` is `True` or executing within 
```

Comment:
re 'PromptedException' - what is Fabric using here?

Reply:
Fabric is calling `SystemExit` instead of raising an exception: https://github.com/mathiasertl/fabric/blob/master/fabric/utils.py#L30-L63

https://github.com/mathiasertl/fabric/blob/master/fabric/state.py#L335